blocks/api/test/function-refs.js

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
Sure, I didn't see the harm in Mocha running it.  It pretty much just skips over it.

`fixtures` directory sounds good, I usually name it `test-data` is that good?